nautobot/extras/migrations/0005_graphqlquery.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.1.8 on 2021-05-05 10:52
```

Comment:
This needs to be renumbered to 0006 (or perhaps just plain regenerated) as it conflicts with the 0005 migration added in #441